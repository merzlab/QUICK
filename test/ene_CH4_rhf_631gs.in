HF  BASIS=6-31G* cutoff=1.0e-9 denserms=1.0e-6  zmake ENERGY DIPOLE CHARGE=0

 C -0.00475919 -0.00238150 -0.07480734
 H -0.90637524  0.51824702 -0.44966396
 H  0.89796302  0.51977168 -0.44485773
 H -0.00288372 -1.04359946 -0.44944522
 H -0.00770827 -0.00393500  1.03405336


