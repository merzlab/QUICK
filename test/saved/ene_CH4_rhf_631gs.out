  *********************************************************************************************************
  **                                                                                                     **
  **           888888888888                                                                              **
  **         8888888888888888                                                                            **
  **      888888888888888888                                                                             **
  **     888888888888888888                                                                              **
  **    888888888888888                                                                                  **
  **   88888888888888888888                               88888                       8888:              **
  **   8888888888888888888888Z                            88888                       8888:              **
  **   888888888888888888888888?                          88888                       8888:              **
  **   8888888888888      8888888                                        888888       8888:              **
  **    88888888888         888888     8888:     88888    88888        888888888I     8888:    888888    **
  **    8888888888           88888:    8888:     88888    88888      $888888888888    8888:   888888     **
  **    I8Z 88888             88888    8888:     88888    88888    .888888     8888   8888: 888888       **
  **    .8Z 88888             88888    8888:     88888    88888    $88888             8888:88888         **
  **     8I 88888      .=88. .88888    8888:     88888    88888    88888              8888888888         **
  **    :8  88888      888888$8888$    8888:     88888    88888    8888O              88888888888        **
  **   ,7   +88888.     8888888888.    8888:     88888    88888    888888             88888O888888       **
  **         $888888:.   .8888888      88888....888888    88888     888888     8888   8888:  888888      **
  **          I8888888888888888888     888888888888888    88888     O888888888888O    8888:   888888     **
  **            O888888888888888888     88888888888888    88888       88888888888$    8888:    888888    **
  **               8888888Z     888      .8888I  88888    88888         8888888       8888:     888888   **
  **                                                                                                     **
  **                                                                                                     **
  **                                         Copyright (c) 2021                                          **
  **                          Regents of the University of California San Diego                          **
  **                                    & Michigan State University                                      **
  **                                        All Rights Reserved.                                         **
  **                                                                                                     **
  **                   This software provided pursuant to a license agreement containing                 **
  **                   restrictions on its disclosure, duplication, and use. This software               **
  **                   contains confidential and proprietary information, and may not be                 **
  **                   extracted or distributed, in whole or in part, for any purpose                    **
  **                   whatsoever, without the express written permission of the authors.                **
  **                   This notice, and the associated author list, must be attached to                  **
  **                   all copies, or extracts, of this software. Any additional                         **
  **                   restrictions set forth in the license agreement also apply to this                **
  **                   software.                                                                         **
  *********************************************************************************************************
                                                                                                           
  Cite this work as:                                                                                       
  Manathunga, M.; Chi, J.; Cruzeiro, V.W.D.; Keipert, K.; Pekurovsky, D.; Mu, D.; Miao, Y.;He, X.; Ayers,K;
  Brothers, E.; GÃ¶tz, A.; Merz,K. M. QUICK-21.03 University of California San Diego, CA and Michigan State
  University, East Lansing, MI, 2021                                                                       
                                                                                                           
  If you have any comments or queries, please send us an email for technical support:                      
  quick.merzlab@gmail.com                                                                                  
                                                                                                           
| TASK STARTS ON: Tue Mar  9 15:01:50 2021
| INPUT FILE :    ene_CH4_rhf_631gs.in
| OUTPUT FILE:    ene_CH4_rhf_631gs.out
| DATE FILE  :    ene_CH4_rhf_631gs.dat
| BASIS SET PATH: /home/mmanathunga/fix109/basis

 @ Read Job And Atom

  KEYWORD=HF  BASIS=6-31G* CUTOFF=1.0E-9 DENSERMS=1.0E-6  ZMAKE ENERGY DIPOLE CHARGE=0                                                                                                                            
 ============== JOB CARD =============
 METHOD = HATREE FOCK
 DIRECT SCF 
 SAD INITAL GUESS 
 USE DIIS SCF
 PRINT ENERGY EVERY CYCLE
 Z-MATRIX CONSTRUCTION
 DIPOLE
 MAX SCF CYCLES =    200
 MAX DIIS CYCLES =   10
 DELTA DENSITY START CYCLE = 1000
 COMPUTATIONAL CUTOFF: 
      TWO-e INTEGRAL   =  0.100E-08
      BASIS SET PRIME  =  0.100E-19
      MATRIX ELEMENTS  =  0.100E-09
      BASIS FUNCTION   =  0.100E-09
 DENSITY MATRIX MAXIMUM RMS FOR CONVERGENCE  =  0.100E-05
 BASIS SET = 6-31G*,  TYPE = CARTESIAN
| BASIS FILE = /home/mmanathunga/fix109/basis/6-31GD.BAS

 @ Finish reading job


 @ Begin SAD initial guess

 For Atom Kind =    1
 ELEMENT = C 
 BASIS FUNCTIONS =   15
 For Atom Kind =    2
 ELEMENT = H 
 BASIS FUNCTIONS =    2

 @ Finish SAD initial guess


 @ Begin Reading Molecular Information


 =========== Molecule Input ==========
 TOTAL MOLECULAR CHARGE  =    0    MULTIPLICITY                =    1
 TOTAL ATOM NUMBER       =    5    NUMBER OF ATOM TYPES        =    2
 NUMBER OF HYDROGEN ATOM =    4    NUMBER OF NON-HYDROGEN ATOM =    1
 NUMBER OF ELECTRONS     =   10

 -- INPUT GEOMETRY -- :
    C          -0.0048      -0.0024      -0.0748
    H          -0.9064       0.5182      -0.4497
    H           0.8980       0.5198      -0.4449
    H          -0.0029      -1.0436      -0.4494
    H          -0.0077      -0.0039       1.0341

 -- DISTANCE MATRIX -- :
              1           2           3           4           5
      1     0.00000
      2     2.09110     0.00000
      3     2.09111     3.40972     0.00000
      4     2.09111     3.40972     3.40973     0.00000
      5     2.09545     3.42332     3.42332     3.42332     0.00000

============== BASIS INFOS ==============
 BASIS FUNCTIONS =   23
 NSHELL =   12 NPRIM  =   44
 JSHELL =   12 JBASIS =   27


 @ End Reading Molecular Information 


 @ Begin Energy Calculation

                                         SCF ENERGY
| ------------------------------------------------------------------------------------------------------------------------
| NCYC       ENERGY         DELTA_E      SCF_TIME  DII_CYC   DII_TIME   O_TIME  DIAG_TIME    MAX_ERR    RMS_CHG    MAX_CHG
| ------------------------------------------------------------------------------------------------------------------------
|  1    -40.253072967      ------         0.014     1        0.00      0.01      0.00    0.2014E+01  0.6760E-01  0.4584E+00
|  2    -40.168071153  -.850018E-01       0.013     2        0.00      0.01      0.00    0.7767E-01  0.1121E-01  0.6281E-01
|  3    -40.191455901  0.233847E-01       0.013     3        0.00      0.01      0.00    0.3204E-01  0.3537E-02  0.3205E-01
|  4    -40.193474791  0.201889E-02       0.013     4        0.00      0.01      0.00    0.7813E-02  0.1277E-02  0.7933E-02
|  5    -40.193675447  0.200656E-03       0.013     5        0.00      0.01      0.00    0.2740E-02  0.3055E-03  0.2775E-02
|  6    -40.193687040  0.115935E-04       0.013     6        0.00      0.01      0.00    0.3147E-03  0.7302E-04  0.4142E-03
|  7    -40.193687527  0.486937E-06       0.013     7        0.00      0.01      0.00    0.2976E-04  0.7461E-05  0.4163E-04
|  8    -40.193687532  0.508535E-08       0.013     8        0.00      0.01      0.00    0.3005E-05  0.1324E-05  0.8751E-05
|  9    -40.193687532  0.651994E-10       0.013     9        0.00      0.01      0.00    0.3858E-06  0.1504E-06  0.5983E-06
| ------------------------------------------------------------------------------------------------------------------------
| REACH CONVERGENCE AFTER   9 CYLCES
| MAX ERROR = 0.385757E-06   RMS CHANGE = 0.150435E-06   MAX CHANGE = 0.598343E-06
| -----------------------------------------------
 ELECTRONIC ENERGY    =   -53.421093911
 CORE_CORE REPULSION  =    13.227406379
 TOTAL ENERGY         =   -40.193687532

 @ End Energy calculation


 @ Begin Zmake conversion

  
 Z-MATRIX:

   ATOM  ELEMENTAL  BOND LENGTH    BOND ANGLE    DIHEDRAL ANGLE
  NUMBER  SYMBOL    (ANGSTROMS)    (DEGREES)       (DEGREES)
     I                 I-NA         I-NA-NB        I-NA-NB-NC      NA   NB   NC

     1      C        0.00000  0     0.00000  0      0.00000  0      0    0    0
     2      H        1.10656  1     0.00000  0      0.00000  0      1    0    0
     3      H        1.10657  1   109.23234  1      0.00000  0      1    2    0
     4      H        1.10657  1   109.23222  1   -119.41952  1      1    2    3
     5      H        1.10887  1   109.70930  1    120.29025  1      1    2    3

 @ End Zmake conversion


 @ Begin Charge and Dipole Calculation

    ATOMIC CHARGES
   ATOM          MULLIKEN            LOWDIN
   C             -0.6536            -0.6150
   H              0.1634             0.1548
   H              0.1634             0.1547
   H              0.1634             0.1532
   H              0.1633             0.1523
   TOTAL         0.0000             0.0000

    DIPOLE (DEBYE)
      X         Y         Z        TOTAL
    0.0000    0.0000   -0.0054    0.0054

 @ End Charge and Dipole Calculation


 @ Output Timing Information

------------- TIMING ---------------
| INITIAL GUESS TIME  =     1.493999000( 92.20%)
| TOTAL SCF TIME      =     0.120892000(  7.46%)
|       TOTAL OP TIME      =     0.119774000(  7.39%)
|             TOTAL 1e TIME      =     0.000996000(  0.06%)
|             TOTAL 2e TIME      =     0.117687000(  7.26%)
|       TOTAL DII TIME      =     0.001072000(  0.07%)
|             TOTAL DIAG TIME    =     0.000449000(  0.03%)
| TOTAL TIME          =     1.620366000
------------------------------------
| Job cpu time:  0 days  0 hours  0 minutes  1.6 seconds.

 @ Finish Output Timing Information

| Normal Termination. Task Finished on: Tue Mar  9 15:01:52 2021
